Sources/PubNubChatComponents/Views/Reaction/Provider/ReactionProvider+Extensions.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  ReactionProvider+Extension.swift
+//  PubNubChat
+//
+//  Created by Maciej Adamczyk on 25/08/2022.
+//
+
+import Foundation
+
+extension ReactionProvider {
```

Comment:
This extension is short enough so we could keep it in the same file where `ReactionProvider` was defined. But it's also OK to have it in a separate file, I don't mind.